data/human/free worlds 2 middle.txt

```diff
@@ -263,13 +265,36 @@ mission "FW Southern Recon 1B (Turret)"
 
 
 
+mission "FW Southern Recon 1B (Captured Gunboat)"
+	landing
+	invisible
+	source
+		government "Free Worlds"
+	to offer
+		has "FW Southern Recon 1B: active"
+		has "ship model (all): Gunboat"
+		or
+			has "outfit (installed): Electron Beam"
+			has "outfit (installed): Electron Turret"
+	on offer
+		dialog `You appear to have captured a Navy Gunboat with the new Electron Beams intact. Uninstall one of the weapons into your cargo, then take off and land on Trinket. (To uninstall the Electron Beam into cargo: select the Gunboat, then select the weapon in the outfitter and press _U, then ctrl + click the Gunboat to deselect it, then select the Electron Beam once more and press _C.)`
```

Comment:
Any reason there's an underscore before the keys?

Reply:
It underlines the letter to mark that it is a key (like when you hold down shift in a menu). I'm not sure whether it is acceptable but I left it in, just waiting on other reviewers' opinions.